### ISSUE TITLE
update build configuration to complete migration from aws s3 to self-hosted

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/globalbioticinteractions#568]
to reduce cost and reduce vendor lock-in, most GloBI services have migrated away from aws s3 . 

A final step is to update the maven wagon configuration .

**Comment by jhpoelen** [comment]
Migration from aws s3 to self-hosted s3-api compatible repository is complete:

1. https://github.com/globalbioticinteractions/maven-s3-wagon was forked from @seahen and changed to allow for multi-part uploads with custom non-aws endpoint (like with a https://min.io endpoint) with configurable chunksize . The changes to the forked maven-s3-plugin include a simplification of the code, added testcases and dependency removal of unmaintained kuali libraries. Also, a more recent version of the aws s3 api is used. 
 
2. for automated reviews, command-line tool ```s3cmd```is now used instead of ```awscli```. The former was easier to configure with non-aws endpoints.

With the migration complete, increased usage/storage of GloBI data products can be managed without having to pay storage rent by the hour and per-per-byte download bandwidth charges. Also, this enables continuous backup schemes without having to rake up even more charges. 

So in short, migrating away from aws s3 has freed the project from significant use changes and will hopefully make it easier to sustain.
